aggregate/all_accessors.py

```diff
@@ -98,6 +99,7 @@
 census_accessors = [
     census_2000_pums_demographics,
     decennial_census_data,
+    ACS_PUMS_economics,
```

Comment:
@SashaWeinstein @td928 Do we wanna rename the accessor to match the file name?

Reply:
yea I think the ACS PUMS vs 2000 naming convention is concise and sufficiently specific. Different than the QOL, housing security and housing prod categories in that there are fewer accessors and so convention matters a little less. Te we do need to figure out whether we want upper or lowercase though

Reply:
That works